packages/app-desktop/gui/ConfigScreen/controls/plugins/PluginBox.tsx

```diff
@@ -137,8 +137,9 @@ export default function(props: Props) {
 	const item = props.item ? props.item : manifestToItem(props.manifest);
 
 	const onNameClick = useCallback(() => {
-		if (!props.item.manifest.homepage_url) return;
-		bridge().openExternal(props.item.manifest.homepage_url);
+		const manifest = item.manifest;
+		if (!manifest.homepage_url) return;
+		bridge().openExternal(manifest.homepage_url);
 	}, [props.item]);
```

Comment:
The dependency should be `item`, not `props.item`. In fact, would you mind memoizing the `item` variable while you're at it? `const item = useMemo(() => { return props.item ? props.item : manifestToItem(props.manifest) }, [props.items, props.manifest])`

Reply:
Also done